### DISCUSSION TITLE
R - Bagaimana cara evaluasi regresi poisson distribution?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#115] [Q&A] [answered]
Saya mau konfirmasi apakah pertimbangan saya salah,
1. Glm dengan family poisson distribution digunakan untuk masalah regressi yang target variablenya count, angkanya sedikit? Betul? 
2. Cara mengevaluasinya apakah bisa dengan menerapkan cara evaluasi yang sama dengan linear regressi? Atau ada perbedaan?
3. Jika mean hampir sama dengan var dan density plot terlihat seperti berikut, apakah dapat dipastikan bahwa nilai goals yang dicetak Erling Haaland merupakan poisson distribution?
![Capture](https://user-images.githubusercontent.com/95233522/170094431-7fb56190-905a-4d58-9171-85b8cf01a970.JPG)

**Comment by ysitta** [comment, marked as answer]
Dear @Hertzspr,

Berikut jawaban untuk poin poin pertanyaan Bapak:

1. Betul, untuk regresi poisson distribution memang cocok untuk data yang discrete. Berikut link yang bisa Bapak jadikan referensi https://algotech.netlify.app/blog/poisson-regression-and-neg-ative-binomial-regression/
2. Terkait evaluasi model, Bapak bisa menggunakan scoring rules. Untuk macam macam scoring rules, bisa merujuk ke sini ya Pak https://stats.stackexchange.com/questions/71720/error-metrics-for-cross-validating-poisson-models
3. Setau saya, untuk poisson distribution nilai mean dan variancenya sama pak.

Apabila Bapak merasa jawaban ini sudah membantu, mohon ditandai answered ya Pak. Apabila ada pertanyaan lanjutan, bisa ditanyakan kembali

**Reply by Hertzspr** [reply]
Terima kasih. 
Mau tanya lagi, kalau var dan meannya tidak sama persis harusnya pakai regresi apa mbak? quasi-poisson? atau negative binomial?
